src/components/templates/Video/HomeVideoList/index.tsx

```diff
@@ -1,46 +1,80 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { getVideoListPopular, getVideoListComplete } from "apis/video";
 import { VideoCard } from "components/modules";
+import { useQuery } from "@tanstack/react-query";
 import * as S from "./style";
 
 const videolist = [
   {
-    title: "title",
-    video: "video",
-    like: 10,
+    videoIdx: 1,
+    title: "뉴진스 민지",
+    link: "https://youtu.be/p1cE9T0CFCQ",
   },
   {
-    title: "title",
-    video: "video",
-    like: 10,
+    videoIdx: 2,
+    title: "엔시티 런쥔",
+    link: "https://youtu.be/mhJfozWlSaM",
   },
   {
-    title: "title",
-    video: "video",
-    like: 10,
+    videoIdx: 3,
+    title: "소녀시대 윤아",
+    link: "https://youtu.be/uoxcux4Scxc",
   },
   {
-    title: "title",
-    video: "video",
-    like: 10,
+    videoIdx: 4,
+    title: "뉴진스 하니",
+    link: "https://youtu.be/lmJPeFW75qQ",
   },
 ];
 
 function HomeVideoList() {
+  const populerQuery = useQuery(["popularVideo"], getVideoListPopular);
```

Comment:
react-query 내부에 state가 내포되어 있어서 popularVideo, newVideo state는 안써도 될 거같아요.

저는 보통 {data, isLoading, error} = useQuery([query key], api 함수) 이렇게 사용합니다.

```suggestion
  const {data:populerQuery, isLoading, error} = useQuery(["popularVideo"], getVideoListPopular);
  
```
이렇게 쓸 것 같습니다.
useQuery가 반환하는 data가 state라고 생각하시면 될 것 같아요!